distros/spring/pom.xml

```diff
@@ -73,6 +73,11 @@
             <artifactId>guava</artifactId>
             <version>28.0-jre</version>
         </dependency>
+        <dependency>
+        <groupId>com.connexta.transformation</groupId>
+        <artifactId>commons-api</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
```

Comment:
❗️ Versions should only be defined in the parent pom dependency management section and using properties. Here you should only reference the artifact without specifying versions. Please do that for all others in here.

Reply:
Austin's PR should be doing the same thing so I think It'd be okay to let his PR do that.

Reply:
As long as the final result is that versions are only defined in the parent pom. We should be in the habit of always doing that anyway.

Reply:
Leaving as is for now to be taken care of in the POST endpoint PR